### PR TITLE
add 404 page and scss for it

### PR DESCRIPTION
**Pull request by pvoros**

### CODE REVIEW

**Review by drianoK — COMMENTED** [review]
Please do not use viewport units for sizing as a scaling value, please change these to px.

File: src/pages/PageNotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styles from './PageNotFound.module.scss';
+
+function PageNotFound() {
+  let option = Math.random() > 0.5;
+  let image = option ? '/images/sherlock.png' : '/images/shakespeare.png';
+  let text = option
```
Comment (bmartin97) [inline src/pages/PageNotFound.jsx]:
```suggestion
  const option = Math.random() > 0.5;
  const image = option ? '/images/sherlock.png' : '/images/shakespeare.png';
  const text = option
```

referring to the style guide, please use const as default, and use let only if you have to reassign the values.

File: src/pages/PageNotFound.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  return (
+    <>
+      <div className={`${styles.pageNotFound}`}>
+        <div className={`${styles.textColumn}`}>
+          <div className={`${styles.errorCode}`}> 404 </div>
+          <div className={`${styles.sorryText}`}> SORRY! </div>
+          <div>{text}</div>
+        </div>
+        <div className={`${styles.imageColumn}`}>
```
Comment (bmartin97) [inline src/pages/PageNotFound.jsx]:
```suggestion
      <div className={styles.pageNotFound}>
        <div className={styles.textColumn}>
          <div className={styles.errorCode}> 404 </div>
          <div className={styles.sorryText}> SORRY! </div>
          <div>{text}</div>
        </div>
        <div className={styles.imageColumn}>
```

I think string interpolations are not needed in those cases. But I'm not sure, please test it if it works or not. Thanks

**Review by bmartin97 — CHANGES_REQUESTED** [review]
Nice job! I wrote some suggestions, that may make the code cleaner, and more aligned with our style guide.

**Review by bmartin97 — APPROVED** [review]

**Review by drianoK — APPROVED** [review]

**Pull request merged**